docs/_templates/breadcrumbs.html

```diff
@@ -1,3 +1,7 @@
+{# Copyright (C) 2016      Robin Schneider <ypid@riseup.net> #}
```

Comment:
Does this highlight mean anything or is it just some strange github thing?

Reply:
It's just the syntax highliter being confused, probably it thinks these are some kind of CSS references.